test/factories/genres.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```

Comment:
this file should be in `spec/factories` instead of  `test/factories`

Reply:
File moved in test/factories